lib/pharos/configuration/host.rb

```diff
@@ -13,6 +13,46 @@ class ResolvConf < Dry::Struct
         attribute :systemd_resolved_stub, Pharos::Types::Strict::Bool
       end
 
+      class Route < Dry::Struct
```

Comment:
Inner classes are usually considered as private to the outer class. In this case `Host::Route` is called from outside and it does not feel right.

Reply:
Right, I didn't notice those other embedded classes from the diff. Maybe `Pharos::Configuration::HostRoute` makes more sense but we can also rethink this later (visual/coding-style issue).